kolena/_utils/endpoints.py

```diff
@@ -97,3 +100,17 @@ def get_model_url(model_id: int) -> str:
 def _get_model_url(client_state: _ClientState, model_id: int) -> str:
     platform_url = _get_platform_url(client_state)
     return f"{platform_url}/models?{urlencode(dict(modelIds=model_id))}"
+
+
+def serialize_models_url(model_id: int, eval_config_id: int) -> str:
+    # Compress and encode to a URL-safe base64 string using LZString
+    return LZString.compressToBase64(
```

Comment:
Is there anything in the standard library we can leverage here to avoid pulling in another dependency?

Reply:
I did not see anything on the std lib, may be we can add this LZString as util class.